config/playground/demo/observability/otel_demo/README.md

```diff
@@ -0,0 +1,97 @@
+
+They are combined of:
+ - [Accounting](https://opentelemetry.io/docs/demo/services/accounting/)
+ - [Ad](https://opentelemetry.io/docs/demo/services/ad/)
+ - [Cart](https://opentelemetry.io/docs/demo/services/cart/)
+ - [Checkout](https://opentelemetry.io/docs/demo/services/checkout/)
+ - [Currency](https://opentelemetry.io/docs/demo/services/currency/)
+ - [Email](https://opentelemetry.io/docs/demo/services/email/)
+ - [Feature Flag](https://opentelemetry.io/docs/demo/services/feature-flag/)
+ - [Fraud Detection](https://opentelemetry.io/docs/demo/services/fraud-detection/)
+ - [Frontend](https://opentelemetry.io/docs/demo/services/frontend/)
+ - [Kafka](https://opentelemetry.io/docs/demo/services/kafka/)
+ - [Payment](https://opentelemetry.io/docs/demo/services/payment/)
+ - [Product Catalog](https://opentelemetry.io/docs/demo/services/product-catalog/)
+ - [Quote](https://opentelemetry.io/docs/demo/services/quote/)
+ - [Recommendation](https://opentelemetry.io/docs/demo/services/recommendation/)
+ - [Shipping](https://opentelemetry.io/docs/demo/services/shipping/)
+ - [Fluent-Bit]() *(Nginx's otel log exported)* 
+ - [Integrations]() *(Pre-canned OpenSearch assets)* 
+ - [DataPrepper]() *(OpenSearch's ingestion pipeline)*
+
+Backend supportive services
+ - [Load Generator]()
+   - See [description]()
+ - [Frontend Nginx Proxy]() *(Replacement for _Frontend-Proxy_)*
+ - [OpenSearch]()
+ - [Dashboards]()
+ - [Prometheus]()
+ - [Feature-Flag]()
+
+### Services Topology
+The next diagram shows the docker compose services dependencies
+
+![](img/docker-services-topology.png)
+---
+
+## Purpose
+The purpose of this demo is to demonstrate the different capabilities of OpenSearch Observability to investigate and reflect your system.
+
+### Ingestion 
+The ingestion capabilities for OpenSearch is to be able to support multiple pipelines:
+  - [Data-Prepper](https://github.com/opensearch-project/data-prepper/) is an OpenSearch ingestion project that allows ingestion of OTEL standard signals using Otel-Collector
+  - [Jaeger](https://opensearch.org/docs/latest/observing-your-data/trace/trace-analytics-jaeger/) is an ingestion framework which has a build in capability for pushing OTEL signals into OpenSearch
+  - [Fluent-Bit](https://docs.fluentbit.io/manual/pipeline/outputs/opensearch) is an ingestion framework which has a build in capability for pushing OTEL signals into OpenSearch
+
+### Integrations -
+The integration service is a list of pre-canned assets that are loaded in a combined manner to allow users the ability for simple and automatic way to discover and review their services topology.
+
+These (demo-sample) integrations contain the following assets:
+ - components & index template mapping
+ - datasources 
+ - data-stream & indices
+ - queries
+ - dashboards
+   
+Once they are loaded, the user can imminently review his OTEL demo services and dashboards that reflect the system state.
```

Comment:
the user can imminently review the OTEL demo services, should we remove 'his' here.